jupyterhub/templates/pod-culler/deployment.yaml

```diff
@@ -1,39 +0,0 @@
-{{- if and .Values.cull.enabled .Values.cull.maxAge -}}
```

Comment:
OH! Note that this culler ran alongside the jupyterhub in-hub-pod service culler since JH 0.9, we had two of them before! I wonder if that caused some issues?

ping @minrk @betatim - in case you have had some unexpected culling behaviors.